heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -181,42 +178,42 @@ export class JestPlugin implements IHeftPlugin<IJestPluginOptions> {
     options?: IJestPluginOptions
   ): Promise<void> {
     const jestLogger: ScopedLogger = heftSession.requestScopedLogger('jest');
-    const buildFolder: string = heftConfiguration.buildFolder;
+    const jestTerminal: Terminal = jestLogger.terminal;
+    jestTerminal.writeLine(`Using Jest version ${getVersion()}`);
 
-    this._jestTerminal = jestLogger.terminal;
-    this._jestTerminal.writeLine(`Using Jest version ${getVersion()}`);
+    const buildFolder: string = heftConfiguration.buildFolder;
 
     // In watch mode, Jest starts up in parallel with the compiler, so there's no
     // guarantee that the output files would have been written yet.
     if (!test.properties.watchMode) {
-      this._validateJestTypeScriptDataFile(buildFolder);
+      await this._validateJestTypeScriptDataFileAsync(buildFolder);
     }
 
-    let jestConfig: string;
+    let jestConfig: IHeftJestConfiguration;
     if (options?.resolveConfigurationModules === false) {
       // Module resolution explicitly disabled, use the config as-is
-      jestConfig = this._getJestConfigPath(heftConfiguration);
-      if (!FileSystem.exists(jestConfig)) {
-        jestLogger.emitError(new Error(`Expected to find jest config file at "${jestConfig}".`));
+      const jestConfigPath: string = this._getJestConfigPath(heftConfiguration);
+      if (!FileSystem.exists(jestConfigPath)) {
```

Comment:
use `existsAsync`?

Reply:
Hmm. I did that but I must have ctrl-z'd it.